source/_integrations/radio_browser.markdown

```diff
@@ -22,6 +22,30 @@ in Home Assistant.
 To start the Radio Browser, in Home Assistant, go to **Media** > **Radio Browser** and select the speaker.
 ![Starting the radio browser](/images/integrations/radio_browser/radio_browser.png)
 
-You can also use the Radio Browser in automations. When creating an automation, use the **Play Media** action to pick a station from the directory. This allows you for example to create
-an automation that starts playing your favorite radio station on your
-Cast devices.
+## Automation
+
+You can also use the Radio Browser in automations. When creating an automation, use the **Play Media** action to pick a station from the directory. This allows you for example, to create
+an automation that starts playing your favorite radio station on your Cast devices. 
+
+The easiest way to find `media_content_id` is to set up the automation via the UI and then switch to the YAML mode. See [Media Player](/integrations/media_player) for more options.
```

Comment:
Considering the text above, I don't understand why you'd need to `media_content_id` at all? The text doesn't provide any reasoning or examples of what to do with it either.